jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ProbeHandlerTest.java

```diff
@@ -220,16 +216,16 @@ public void getTCPProbeWithNonHTTPURLAndStringPortTest() {
 
         probe = probeHandler.getProbe(probeConfig);
         //assertion
-        assertNotNull(probe);
-        assertNull(probe.getHttpGet());
-        assertNotNull(probe.getTcpSocket());
-        assertNull(probe.getExec());
-        assertEquals("httpPort", probe.getTcpSocket().getPort().getStrVal());
-        assertEquals("www.healthcheck.com",probe.getTcpSocket().getHost());
-        assertEquals(5,probe.getInitialDelaySeconds().intValue());
-        assertEquals(5,probe.getTimeoutSeconds().intValue());
-        assertEquals(3, probe.getFailureThreshold().intValue());
-        assertEquals(1, probe.getSuccessThreshold().intValue());
+        assertThat(probe).isNotNull();
+        assertThat(probe.getHttpGet()).isNull();
+        assertThat(probe.getTcpSocket()).isNotNull();
+        assertThat(probe.getExec()).isNull();
+        assertThat(probe.getTcpSocket().getPort().getStrVal()).isEqualTo("httpPort");
+        assertThat(probe.getTcpSocket().getHost()).isEqualTo("www.healthcheck.com");
+        assertThat(probe.getInitialDelaySeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getTimeoutSeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getFailureThreshold().intValue()).isEqualTo(3);
+        assertThat(probe.getSuccessThreshold().intValue()).isEqualTo(1);
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.